Documentation/ApiOverview/SiteHandling/BaseVariants.rst

```diff
@@ -55,6 +55,9 @@ Example
       -
         base: 'https://testing.example.com/'
         condition: 'applicationContext == "Testing/Paris"'
+      -
+        base: '%env("TYPO3_BASE")%'
+        condition: 'getenv("TYPO3_BASE")'
```

Comment:
Why not use `%env()%` or `getenv()` for both?

Reply:
`getenv()` doesn't seem to work within `base`.
I tried it and TYPO3 couldn't find the site.

Also `getenv()` is the only documented way on that page for `condition`, therefore I kept that. And I also didn't try whether `%env()%` will work within the condition.

I guess only `condition` is passed through the expression language, which resolves the functions. `%env()%` might be related to the yaml parser and work in both places?

I didn't check the source, just added what I figured out, to look it up again.

Not sure if those explanations help. I can adjust the PR if you wish.

Reply:
I'm fine with anything, as long it is either 1) obvious or 2) well-explained. Just like this it's just confusing so I'd vote for using `%env()%` for `condition` too. You are right that this is a YAML parser feature which would in the worst case lead to `condition: ''` which would evaluate to `false`.

Reply:
I've added a cross-reference to existing documentation for `%env()%`.
I hope that matches 2, well explained.

But I can also give `%env()%` in both places a try. I don't think that is a best practice or used by many anyway. I can also add the example in a separate section, but wanted to keep changes at a minimum.

Reply:
I just learned the hard way that `condition: '%env(TYPO3_BASE)%'` does not work. However, with `'getenv("TYPO3_BASE")'` the base is applied.